records/2023-01-26.bounce-rate-automated-emergency-brake.md

```diff
@@ -0,0 +1,83 @@
+
+#### Minimum volume threshold
+Low-volume senders won't have a large impact on the bounce rate, and notifying them directly may lead to many support calls. In order to limit this, only notify services that are sending more than [bounce-rate-service-volume-minimum] emails in the [bounce-rate-window].  This minimum could either be a fixed number or a percentage of the total number of emails sent by all services.
+
+### Suspend a service when it reaches the [bounce-rate-threshold]
+When a service reaches the [bounce-rate-threshold], suspend their service to prevent them from sending any more emails. Notify the service owner that their service has been suspended and provide them with a link to the service's dashboard where they can see their bounce rate and take action to reduce it.
+
+:question: _How to decide when it is appropriate to unsuspend a service?_
+
+### Add bounce rate information to the service dashboad
+Add the bounce rate to the service dashboard so that service owners can see their bounce rate and proactively read the guidance on it so they can prevent their bounce rate from rising in the first place.
+
+### Provide bounce rate guidance
+Provide guidance to service owners to explain bounce rate concepts and how they apply to their service.  Provide guidance on how services can keep their bounce rate low. 
+
+### bounce-rate-threshold calculation options
+When calculating a service's bounce rate, we can either consider them in isolation or in aggregate with other services.  The bounce rate is calculated as a percentage of the total number of emails sent.  If we consider a service in isolation, then the bounce rate will be calculated as a percentage of the total number of emails sent by that service.  If we consider a service in aggregate, then the bounce rate will be calculated as a percentage of the total number of emails sent by all services.
+
+#### Option 1: Consider a service in isolation
+Considering a service in isolation would mean calculating the bounce rate solely based on an individual services activity.  The benefit of this is that the calculation would be straight-forward and easy to understand by the service.  The downside is that even if a particular service has a high bounce rate, it may not have a large impact on the overall bounce rate. 
+
+#### Option 2: Consider a service in aggregate
+Considering a service in aggregate would mean calculating the bounce rate based on the total number of emails sent by all services.  The benefit of this is that the calculation would be more accurate and would reflect the impact of a particular service on the overall bounce rate.  The downside here is that the calculation would be more complex and may be difficult to understand by the service, since for example even if they had an extremely high bounce rate, we might not indicate it as problematic if it has little impact on the overall bounce rate.
```

Comment:
OK I see what you are going for here, trying to allow for a higher bounce rate for an individual services if the total volume of notification can amortize it. 

You are right in saying this will be a difficult rule to explain to our users. We'd have to explain this in our content and I am not sure it is best to expose influencing metrics transparently to our users (e.g. global bounce rate).

This feels overly complex and potentially, could lead to use cases where a service that has a bounce rate of only 5% to get suspended because the global bounce rate was previously boosted up by another service that was deemed OK to reach a 20% bounce rate with amortization. I think there might be ways to enable amortization, but not in a dynamic/unplanned manner, which could affect the services ecosystem.

Reply:
Agreed, after considering this some more, this won't provide much additional benefit for us in the end, and its best to keep our each user's individual bounce rate low no matter what any other user is doing on the platform.  I'll re-work this and simplify it.